src/storage/storage.rs

```diff
@@ -8,7 +12,14 @@ use super::secret::SecretEntity;
 pub struct Storage {
     pub client: Client,
     pub db: Database,
-    pub secret_collection: Collection<SecretEntity>
+    pub secret_collection: Collection<SecretEntity>,
+}
+
+#[derive(Copy, Clone)]
+pub struct AttemptCountRule {
```

Comment:
Since this struct is only being used at the API level, I think we can move it out of the storage code.

It's currently used on our `main`, and in our `handlers/secret` file. I think we can move it to the handlers file, the main file can read that too.

Reply:
👍